lms/envs/common.py

```diff
@@ -349,6 +349,9 @@
     'ENABLE_COURSEWARE_SEARCH': False,
 }
 
+# Add Theme values here
+THEME_VALUES = {}
```

Comment:
How about calling this `THEME_CONFIGURATION`? `THEME_VALUES` is slightly more vague; all variables are "values" :)